geo/src/algorithm/bool_ops/mod.rs

```diff
@@ -65,19 +64,13 @@ impl<T: GeoFloat> BooleanOps for MultiPolygon<T> {
         let mut bop = Op::new(op, self.coords_count() + other.coords_count());
         bop.add_multi_polygon(self, true);
         bop.add_multi_polygon(other, false);
-        let rings = bop.sweep();
-        assemble(rings).into()
+        bop.sweep()
     }
 }
 
 mod op;
 use op::*;
-
-mod rings;
-use rings::{Ring, Rings};
-
-mod laminar;
-use laminar::*;
```

Comment:
The original RFC references this module:

> The region enclosed by the rings form a [Laminar Set]. We re-use the
planar-sweep to again detect the inclusion relations of the rings, and
assemble as polygons.  Ref `geo/src/algorithm/bool_ops/laminar.rs`.

I don't do much work with RFCs. Does it make sense to try to keep the RFC up to date with the current state of things? Or is it intended to be "locked in time" as things were originally implemented?